src/ripple/app/main/Application.cpp

```diff
@@ -1705,6 +1706,39 @@ void ApplicationImp::addTxnSeqField ()
     tr.commit ();
 }
 
+void ApplicationImp::addValidationSeqFields ()
+{
+    if (schemaHas(getLedgerDB(), "Validations", 0, "LedgerSeq", m_journal))
+    {
+        assert(schemaHas(getLedgerDB(), "Validations", 0, "InitialSeq", m_journal));
+        return;
+    }
+
+    JLOG(m_journal.warning) << "Validation sequence fields are missing";
+    assert(!schemaHas(getLedgerDB(), "Validations", 0, "InitialSeq", m_journal));
+
+    auto& session = getLedgerDB().getSession();
+
+    soci::transaction tr(session);
+
+    JLOG(m_journal.info) << "Altering table";
+    session << "ALTER TABLE Validations            \
+        ADD COLUMN LedgerSeq       BIGINT UNSIGNED;";
```

Comment:
Heh. Nevermind - I just saw the string split over two lines and thought to suggest raw strings. Fine as is.